### PR TITLE
feat: packet resolver from mapleserver2

### PR DESCRIPTION
**Pull request by AngeloTadeucci**
New wiki entry as well https://github.com/AngeloTadeucci/Maple2/wiki/Packet-Resolver

### CODE REVIEW

File: Maple2.Server.Core/Packets/Helper/ErrorParserHelper.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
+﻿using System.Diagnostics;
```
Comment (Zintixx) [inline Maple2.Server.Core/Packets/Helper/ErrorParserHelper.cs:1]: Can we put this in Maple2.Tools?

Comment (AngeloTadeucci) [reply Maple2.Server.Core/Packets/Helper/ErrorParserHelper.cs:1]: nah, it uses stuff from core and core isn't references in tools

Comment (Zintixx) [reply Maple2.Server.Core/Packets/Helper/ErrorParserHelper.cs:1]: okay let's put it under Util/Helpers then?

Comment (AngeloTadeucci) [reply Maple2.Server.Core/Packets/Helper/ErrorParserHelper.cs]: didn't notice it was inside /packets lul

**Comment by kOchirasu** [comment]
I don't even remember how to use this but you know you can do overrides as well right? I didn't see anything mentioned in the docs. The syntax was something like `(offset, type, value)`

So for example: `/debug resolve opcode (20, 4, 1)` would set the int(4) at offset(20) to value(1).

**Comment by AngeloTadeucci** [comment]
> I don't even remember how to use this but you know you can do overrides as well right? I didn't see anything mentioned in the docs. The syntax was something like `(offset, type, value)`
> 
> So for example: `/debug resolve opcode (20, 4, 1)` would set the int(4) at offset(20) to value(1).

think when I reworked I remove that functionality

**Review by Zintixx — APPROVED** [review]

**Pull request merged**